master/internal/db/postgres.go

```diff
@@ -59,7 +59,7 @@ func initTheOneBun(db *PgDB) {
 	}
 
 	// This will print every query that runs.
-	// theOneBun.AddQueryHook(bundebug.NewQueryHook(bundebug.WithVerbose(true)))
+	theOneBun.AddQueryHook(bundebug.NewQueryHook(bundebug.WithVerbose(true)))
```

Comment:
!

Reply:
lol. I would like a different devcluster default with `-race` and a bunch of debug logging like this.

Reply:
probably will do it this week as part of turning on -race for e2es

Reply:
lol, I added a linter rule which errors if it uncommented

when you make it configurable you can delete the rule